src/environmentd/src/bin/environmentd/main.rs

```diff
@@ -800,7 +802,10 @@ fn run(mut args: Args) -> Result<(), anyhow::Error> {
             .try_into()
             .expect("must fit"),
     );
+    let span = span.exit();
+    let id = span.context().span().span_context().trace_id();
     drop(span);
+    println!("environmentd run span id: {id}");
```

Comment:
Can you move this to line 824 and emit it as "  Root trace ID:", to follow the formatting for the other informational output at boot? (And it is a trace ID and not a span ID, right?)

Reply:
done